spec/new-productions.htm

```diff
@@ -9,6 +9,18 @@
   </emu-grammar>
 </emu-clause>
 
+<emu-clause id="public-field-definition-early-errors">
+  <h1>Static Semantics: Early Errors</h1>
+    PublicFieldDefinition :
+      PropertyName[?Yield] Initializer?
+  <emu-grammar>
+  </emu-grammar>
+  <emu-alg>
+    1. It is a SyntaxError if |Initializer| contains |NewTarget|.
```

Comment:
OK, I see there are a couple Contains usages, but it's mostly contains. It doesn't matter which one is used; probably the spec could use an editorial change to normalize.